.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: objective-c
 osx_image: xcode7.1
 
+before_install:
+  - brew update
+  - brew install swiftlint 
+
 install:
-    - gem install xcpretty
-    - gem install cocoapods
-    - pod install
+  - gem install xcpretty
+  - gem install cocoapods
+  - pod install
```

Comment:
Can you move that part of the script to the before_install before to merge this PR?